docs/src/content/docs/ru/guides/sidebar.mdx

```diff
@@ -0,0 +1,524 @@
+Например, со следующей конфигурацией:
+
+```js
+starlight({
+	sidebar: [
+		{
+			label: 'Руководства',
+			// Автоматически генерировать группу ссылок для директории "руководства".
+			autogenerate: { directory: 'guides' },
+		},
+	],
+});
+```
+
+И следующей структурой файлов:
+
+<FileTree>
+
+- src/
+  - content/
+    - docs/
+      - guides/
+        - components.md
+        - i18n.md
+        - advanced/
+          - project-structure.md
+
+</FileTree>
+
+Следующая боковая панель будет сгенерирована:
+
+<SidebarPreview
+	config={[
+		{
+			label: 'Руководства',
+			items: [
+                { label: 'Компоненты', link: '/ru/guides/components/' },
+                { label: 'Интернационализация (i18n)', link: '/ru/guides/i18n/' },
+				{
+					label: 'дополнительно',
+					items: [
+						{ label: 'Структура проекта', link: '/ru/guides/project-structure/' },
+					],
+				},
+			],
+		},
+	]}
+/>
+
+#### Настройка сгенерированных ссылок через метаданные
+
+Используйте [поле `sidebar`](/ru/reference/frontmatter/#sidebar) в метаданных страниц для настройки автоматически генерируемых ссылок.
+
+Параметры в метаданных для боковой панели позволяют [установить метку](/ru/reference/frontmatter/#label)
+или добавить [значок](/ru/reference/frontmatter/#badge) к ссылке, [скрыть](/ru/reference/frontmatter/#hidden)
+ссылку из боковой панели или определить [вес сортировки](/ru/reference/frontmatter/#order).
+
+```md
+---
+title: Моя страница
+sidebar:
+  # Установить текст для ссылки
+  label: Текст в боковой панели
+  # Установить порядок для ссылки (меньшие числа отображаются выше)
+  order: 2
+  # Добавить значок к ссылке
+  badge:
+    text: New
+    variant: tip
+---
+```
+
+Автоматически созданная группа, включающая страницу с вышеуказанными метаданными, сгенерирует следующую боковую панель:
+
+<SidebarPreview
+	config={[
+		{
+			label: 'Руководства',
+			items: [
+				{ label: 'Страница', link: '#' },
+				{
+					label: 'Текст в боковой панели',
+					link: '#',
+					badge: { text: 'New', variant: 'tip' },
+				},
+				{ label: 'Другая страница', link: '#' },
+			],
+		},
+	]}
+/>
+
+:::note[Заметка]
+Конфигурация `sidebar` в метаданных используется только для автоматически генерируемых ссылок и будет проигнорирована для вручную определенных ссылок.
+:::
+
+## Значки
+
+Ссылки также могут включать свойство `badge` для отображения значка рядом с текстом ссылки.
+
+```js
+starlight({
+	sidebar: [
+		{
+			label: 'Руководства',
+			items: [
+				// Ссылка со значком "New".
+				{
+					label: 'Компоненты',
+					link: '/ru/guides/components/',
+					badge: 'New',
+				},
+			],
+		},
+	],
+});
+```
+
+Конфигурация выше создаст следующую боковую панель:
+
+<SidebarPreview
+	config={[
+		{
+			label: 'Руководства',
+			items: [
+				{
+					label: 'Компоненты',
+					link: '/ru/guides/components/',
+					badge: { text: 'New', variant: 'default' },
```

Comment:
The preview should be updated too to translate "New".

Reply:
done